src/App.js

```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ]  It would be better if the Function app is class-based. Kindly fix this minor issue. 😄 🔧 

```suggestion
class App extends Component {
  return (
```